templates/projects/performancestats.html

```diff
@@ -0,0 +1,19 @@
+{% extends "projects/base.html" %}
+{% set active_link = "performancestats" %}
+{% set section = _('Performance Statistics') %}
+
+{% block projectcontent %}
+
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.17/d3.min.js"></script>
```

Comment:
do you think is better having the D3 from cdn instead of using npm package?

Reply:
Thanks, I'll pull it from npm